inference-engine/src/mkldnn_plugin/nodes/mkldnn_cum_sum_node.cpp

```diff
@@ -45,11 +41,11 @@ MKLDNNCumSumNode::MKLDNNCumSumNode(const std::shared_ptr<ngraph::Node>& op, cons
     if ((getOriginalInputsNumber() != numOfInputs && getOriginalInputsNumber() != (numOfInputs - 1)) || getOriginalOutputsNumber() != 1)
         IE_THROW() << errorPrefix << " has incorrect number of input/output edges!";
 
-    const auto &dataShape = op->get_input_shape(CUM_SUM_DATA);
-    if (dataShape.size() < 1) {
-        IE_THROW() << errorPrefix << " doesn't support 'data' input tensor with rank: " << dataShape.size();
+    const auto &dataShape = getInputShapeAtPort(CUM_SUM_DATA);
+    if (dataShape.getRank() < 1) {
+        IE_THROW() << errorPrefix << " doesn't support 'data' input tensor with rank: " << dataShape.getRank();
     }
-    numOfDims = dataShape.size();
+    numOfDims = dataShape.getRank();
```

Comment:
minor
maybe move above and use instead `dataShape.getRank()` in comparasion

Reply:
Fixed, thank you